parser-typechecker/unison-parser-typechecker.cabal

```diff
@@ -191,6 +191,7 @@ library
     configurator,
     cryptonite,
     directory,
+    either,
```

Comment:
What’s the license on this? Just checking.

Reply:
we already include it in the -core project, so it's already documented